src/cmd/Cargo.toml

```diff
@@ -46,7 +46,11 @@ workspace-hack = { path = "../workspace-hack" }
 task_stats_alloc = { path = "../utils/task_stats_alloc" }
 
 [target.'cfg(unix)'.dependencies]
-tikv-jemallocator = { git = "https://github.com/yuhao-su/jemallocator.git", features = ["profiling", "stats", "unprefixed_malloc_on_supported_platforms"], rev = "a0911601bb7bb263ca55c7ea161ef308fdc623f8" }
+tikv-jemallocator = { git = "https://github.com/risingwavelabs/jemallocator.git", features = [
```

Comment:
What about making it a workspace dependency?

https://github.com/risingwavelabs/risingwave/blob/88418920d3f350d19cd6d674e92681357bcbd271/Cargo.toml#L62

Reply:
Let us do it in future PRs. cc. @yuhao-su